src/core/gui/XournalView.cpp

```diff
@@ -287,6 +289,22 @@ auto XournalView::onKeyPressEvent(GdkEventKey* event) -> bool {
         return true;
     }
 
+    // Switch color on number key
+    auto colors = control->getWindow()->getToolMenuHandler()->getColorToolItems();
```

Comment:
nit: when using auto, it helps readability to be explicit about whether the type is a pointer or reference.
```suggestion
    auto *colors = control->getWindow()->getToolMenuHandler()->getColorToolItems();
```